docs/docs/guides/integrate/token-introspection/private-key-jwt.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: JSON Web Token profile
+---
+
+This is a guide on how to secure your API using [JSON Web Token (JWT) profile (recommended)](https://zitadel.com/docs/apis/openidoauth/authn-methods#client-secret-basic).  
+
+## Register the API in ZITADEL and generate private and public keys 
+
+1. Go to your project and click on the **New** button as shown below. 
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-1.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+2.  Give a name to your application (Test API is the name given below) and select type **API**. 
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-2.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+3. Select **JWT** as the authentication method and click **Continue**.  
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-3.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+4. Now review your configuration and click **Create**.   
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-4.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+5. You will now see the API’s **Client ID**. You will not see a client secret because we are using a private JWT key.   
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-5.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+6. Next, we must create the key pairs. Click on **New**.   
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-6.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+7. Select **JSON** as the type of key. You can also set an expiration time for the key or leave it empty. Click on **Add**.   
+
+<img
+    src="/docs/img/guides/integrate/token-introspection-jwt-profile-7.png"
+    width="75%"
+    alt="Register the API"
+/>
+
+8. Download the created key by clicking the **Download** button and then click **Close**.   
+
+<img
```

Comment:
This image looks broken:
<img width="556" alt="image" src="https://github.com/zitadel/zitadel/assets/38692350/5cd6904f-cd72-45d3-9382-6abddcdf3917">

Reply:
Fixed it.